test/LRPTests/APITests.cpp

```diff
@@ -0,0 +1,157 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+#include <TestDef.h>
+#include <TlHelp32.h>
+#include <NotificationsLongRunningProcess_h.h>
+
+using namespace WEX::Common;
+using namespace WEX::Logging;
+using namespace WEX::TestExecution;
+
+using namespace winrt::Windows::Foundation;
+using namespace winrt::Windows::System;
+
+namespace Test::LRP
+{
+    static const PCWSTR c_processName = L"TAEF.exe";
+    static const PCWSTR c_appId = L"toastAppId";
+
+    class LRPTests
+    {
+
+    public:
+        BEGIN_TEST_CLASS(LRPTests)
+            TEST_CLASS_PROPERTY(L"Description", L"Windows App SDK Push Notifications Long Running Process tests")
+            TEST_CLASS_PROPERTY(L"ThreadingModel", L"MTA")
+            TEST_CLASS_PROPERTY(L"RunAs:Class", L"RestrictedUser")
+        END_TEST_CLASS()
+
+        winrt::com_ptr<INotificationsLongRunningPlatform> GetNotificationPlatform()
+        {
+            winrt::com_ptr notificationPlatform{ winrt::try_create_instance<INotificationsLongRunningPlatform>(_uuidof(NotificationsLongRunningPlatform), CLSCTX_ALL) };
+            VERIFY_IS_NOT_NULL(notificationPlatform.get());
```

Comment:
Optional: wondering if it is succeeded assign it to a class variable? that way you don't have to keep creating and destroying an instance of it. Let me know.

Reply:
As long as a client has a reference to the notificationPlatform, the LRP will stay alive. We don't want this behavior for LaunchLRP_FromStartupTask since we are verifying the LRP is dead then comes alive in the test from the startuptask.exe. If it wasn't for this test, we could use a member variable.

Reply:
Thanks for the clarification.